lib/jsonapi-serializers/serializer.rb

```diff
@@ -286,7 +285,6 @@ def self.serialize(objects, options = {})
         'data' => primary_data,
       }
       result['meta'] = options[:meta] if options[:meta]
-      result['errors'] = options[:errors] if options[:errors]
```

Comment:
Sorry, by "deprecate" I mean leave it in so we don't break backward's compatibility. :) Can you restore these lines and add a comment like "Deprecated: use serialize_errors instead."